conf/modules.config

```diff
@@ -1126,37 +1140,68 @@ if (!params.skip_multiqc) {
 }
 
 //
-// Salmon pseudo-alignment options
+// Salmon/ Kallisto pseudo-alignment options
 //
 
-if (!params.skip_pseudo_alignment && params.pseudo_aligner == 'salmon') {
+if (params.pseudo_aligner == 'salmon') {
     process {
-        withName: '.*:QUANTIFY_SALMON:SALMON_QUANT' {
+        withName: '.*:QUANTIFY_PSEUDO:SALMON_QUANT' {
             ext.args   = params.extra_salmon_quant_args ?: ''
             publishDir = [
                 path: { "${params.outdir}/${params.pseudo_aligner}" },
                 mode: params.publish_dir_mode,
                 saveAs: { filename -> filename.equals('versions.yml') || filename.endsWith('_meta_info.json') ? null : filename }
             ]
         }
+    }
+}
+
+if (params.pseudo_aligner == 'kallisto') {
+    process {
+        withName: '.*:QUANTIFY_PSEUDO:KALLISTO_QUANT' {
+            ext.args = {
+                [
+                    meta.single_end ?
+                        "--single " +
+                        "--fragment-length ${meta.fragment_len ?: params.kallisto_default_fraglen} " +
+                        "--sd ${meta.fraglen_se ?: params.kallisto_default_fraglen_se}" : '',
+                    params.kallisto_quant_strandedness ? '--' + params.kallisto_quant_strandedness:
+                        (meta.strandedess == 'forward' ? '--fr-stranded':
+                            (meta.strandedness == 'reverse' ? '--rf-stranded' : '')
+                        ),
```

Comment:
Add this logic to the module like we do for Salmon? Will make the module more portable without needing a config.

Reply:
That will require me to add assumptions to the module on `meta`, right? I know the salmon module does that, but I've been told off for it in the past.

Reply:
Checking with maintainers, view is that we should actually nuke the older pattern in Salmon

Reply:
Waiting on https://github.com/nf-core/modules/pull/4309 and maintainer feedback